deploy/official-postgresql/templates/clusterdefinition.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterDefinition
+metadata:
+  name: official-postgresql
+  labels:
+    {{- include "official-postgresql.labels" . | nindent 4 }}
+spec:
+  type: postgresql
+  connectionCredential:
+    username: postgres
+    password: "$(RANDOM_PASSWD)"
+    endpoint: "$(SVC_FQDN):$(SVC_PORT_tcp-postgresql)"
+    host: "$(SVC_FQDN)"
+    port: "$(SVC_PORT_tcp-postgresql)"
+  componentDefs:
+    - name: postgresql
+      workloadType: Replication
+      characterType: postgresql
+      probes:
+        roleProbe:
+          failureThreshold: {{ .Values.roleProbe.failureThreshold }}
+          periodSeconds: {{ .Values.roleProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.roleProbe.timeoutSeconds }}
+      configSpecs:
+        - name: official-postgresql-configuration
+          templateRef: official-postgresql14-configuration
+          constraintRef: official-postgresql14-cc
+          keys:
+            - postgresql.conf
+          namespace: {{ .Release.Namespace }}
+          volumeName: postgresql-config
+          defaultMode: 0444
+      scriptSpecs:
+        - name: official-postgresql-scripts
+          templateRef: official-postgresql-scripts
+          namespace: {{ .Release.Namespace }}
+          volumeName: scripts
+          defaultMode: 0555
+      service:
+        ports:
+          - name: tcp-postgresql
+            port: 5432
+            targetPort: tcp-postgresql
+      volumeTypes:
+        - name: data
+          type: data
+      podSpec:
+        securityContext:
+          runAsUser: 0
+          fsGroup: 103
+          runAsGroup: 103
+        containers:
+          - name: postgresql
+            imagePullPolicy: {{ default .Values.image.pullPolicy "IfNotPresent" }}
+            securityContext:
+              runAsUser: 0
+            command:
+              - /kb-scripts/setup.sh
+            readinessProbe:
```

Comment:
readinessProbe is not necessary here and could adversely affect the headless service.

Reply:
/done